packages/javascript/bh-shared-ui/src/components/ExploreTable/ManageColumnsComboBox/ManageColumnsListItem.tsx

```diff
@@ -37,7 +37,7 @@ const ManageColumnsListItem = ({ isSelected, item, onClick, itemProps }: ManageC
             e.stopPropagation();
             onClick(item);
         }}>
-        <button className="w-full text-left flex justify-between items-center'">
+        <div className="w-full text-left flex justify-between items-center'">
```

Comment:
This shouldnt change functionality but will address a console/test log warning. The checkbox component is a button and click events will still bubble up to the ListItem onClick, I believe this will be easier for screen readers compared to the nested button, but please correct me if im wrong on that!!